packages/core/saved-objects/core-saved-objects-api-server/src/apis/update.ts

```diff
@@ -34,6 +34,13 @@ export interface SavedObjectsUpdateOptions<Attributes = unknown> extends SavedOb
   retryOnConflict?: number;
   /** {@link SavedObjectsRawDocParseOptions.migrationVersionCompatibility} */
   migrationVersionCompatibility?: 'compatible' | 'raw';
+  /**
+   * By default, update will merge the provided attributes with the ones present on the document
+   * (performing a standard partial update). Setting this option to `true` will change the behavior, performing
+   * a "full" update instead, where the provided attributes will fully override the existing ones.
+   * Defaults to `false`.
+   */
+  overrideAttributes?: boolean;
```

Comment:
As discussed, `mergeAttributes: true` seems more appropriate.

Reply:
Done in https://github.com/elastic/kibana/pull/183267/commits/58574478a0473f90c7c41e642ce1e4a9e70ee902